src/NTH.Windows.Forms/NTH.Windows.Forms/NthListViewGroup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace NTH.Windows.Forms
+{
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+    public struct NthListViewGroup
```

Comment:
Actually the current accessor isn't necessary as the NthListViewGroup-structure is only accessed by the relating NthListView-class. I will change it to "internal". The fields' declaration should be ok then.

I will move it into NativeTyes-namespace.